CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+<!--
+SPDX-FileCopyrightText: 2023 Foundation Devices, Inc. <hello@foundationdevices.com>
+
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+## Head
+- Added changelog #581
```

Comment:
The `#581` will be confused with GitHub PRs.  Suggest putting the full PASS1-581 instead.

Reply:
Should I do this for each issue number?